ocaml/xapi/xapi_pbd.ml

```diff
@@ -213,7 +200,7 @@ let unplug ~__context ~self =
 
 let destroy ~__context ~self =
   if Db.PBD.get_currently_attached ~__context ~self
-  then raise (Api_errors.Server_error(Api_errors.operation_not_allowed, ["PBD is currently attached"]));
+  then raise Api_errors.(Server_error (operation_not_allowed, ["PBD is currently attached"]));
```

Comment:
There are some unrelated cleanups in these commits. Thats fine, but could you move them to a separate commit?

Reply:
If you want to move these into a separate commit you can do so by doing a `git reset HEAD~1` and then `git add -p` and choose which changes to add to this commit, and then whats left can be in a next commit.